inc/classes/admin/settings/class-page.php

```diff
@@ -888,21 +888,46 @@ private function media_section() {
 			]
 		);
 
-		$rocket_maybe_disable_lazyload_plugins = [];
+		$disable_images_lazyload  = [];
```

Comment:
We touched this file. It should be moved into our new architecture, renamed, and the old deprecated.